src/Desktop/Atlas/1. Install Software/HOW TO.txt

```diff
@@ -0,0 +1 @@
+To install software, you can either run Install Choco or Install Scoop, you have to install one to install software from this part
```

Comment:
Maybe add that Chocolatey is recommended for web browsers and other non-portable applications?

Reply:
Added, thanks